label_sleuth/app.py

```diff
@@ -409,15 +410,19 @@ def get_document_elements(workspace_id, document_uri):
     :param document_uri:
     :request_arg category_id:
     """
+
+    size = int(request.args.get('size', 100))
+    start_idx = int(request.args.get('start_idx', 0))
```

Comment:
maybe a larger default value can be used?
we can use 500 as the default of main_panel_elements_per_page

Reply:
Good idea, I will use the configuration variable value instead